RELEASE_NOTES

```diff
@@ -1,6 +1,20 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.1.1 (14 november 2023)
```

Comment:
Nitpicking: let's fix this (see also https://www.grammarly.com/blog/capitalization-the-days-of-the-week-the-months/), same for `v0.1.0` release date

```suggestion
v0.1.1 (14 November 2023)